README.md

```diff
@@ -25,6 +25,10 @@ export default {
   name: 'MyModule',
   plugins: [
     resolve({
+      // use "esnext" if possible
```

Comment:
A quick description of the `esnext` would be useful. `package.json` entries are confusing enough.

Maybe something like:
```js
  // use "esnext" field for ES6 modules with non-transpiled ES6+ features if possible
  // – see http://2ality.com/2017/06/pkg-esnext.html
  esnext: true,  // Default: false

  // use "module" field for ES6 modules if possible
  // – see http://2ality.com/2017/06/pkg-esnext.html
  module: true,  // Default: true
```